web/client/components/cookie/Cookie.jsx

```diff
@@ -128,7 +128,11 @@ class Cookie extends React.Component {
         this.props.onMoreDetails(!this.props.seeMore);
     }
     accept = () => {
-        localStorage.setItem("cookies-policy-approved", true);
+        try {
+            localStorage.setItem("cookies-policy-approved", true);
```

Comment:
Shouldn't we use the api here?

Reply:
Still in progress?

Reply:
yes is a wip
i need to provide changes if the api is good